src/components/Organization/index.tsx

```diff
@@ -109,11 +109,11 @@ export default function OrganizationView({ organization }: OrganizationViewProps
             <h2>{organization?.name}</h2>
           </Grid>
           <Grid item xs={4} className={classes.titleButtonContainer}>
-            <Button
-              label={isMobile ? strings.EDIT : strings.EDIT_ORGANIZATION}
-              priority='secondary'
-              onClick={goToEditOrganization}
-            />
+            {isMobile ? (
+              <Button icon='iconEdit' priority='secondary' onClick={goToEditOrganization} />
+            ) : (
+              <Button icon='iconEdit' label={strings.EDIT} priority='secondary' onClick={goToEditOrganization} />
+            )}
```

Comment:
Do we have a preference on how to handle cases like this - where the only difference is the label?
```
{ isMobile ? <mobile markup> : <desktop markup>}
```
vs
```
<markup label={isMobile ? '' : string} >
```

We have been mostly using the latter in the codebase.. however, the former does express intent clearly.

Just a question for all. Not requesting any changes at this point.

Reply:
What I try to do is that if only one attribute is different between the two elements option 2 is better, but if we have more than one attributed changed (for example label, priority, icon) it's better to create two elements

Reply:
I like the latter myself, it's more concise and also gets the intent across I think. Certainly if you have a lot of props conditional on `isMobile` the former would be the way to go. If you had other props that were conditional on other variables you might get into some kinda nasty nesting, but at that point you might want to rethink inlining the logic and maybe break out into another component 🤷

Reply:
I am leaning towards the former now that I see it in code. We have an impasse 😱 - we could evolve our codestyle over time maybe, now that the team is growing?